themes/src/main/resources-community/theme/base/email/messages/messages_fa.properties

```diff
@@ -39,14 +39,9 @@ requiredAction.VERIFY_EMAIL=تأیید ایمیل
 requiredAction.CONFIGURE_RECOVERY_AUTHN_CODES=کدهای بازیابی را ایجاد کنید
 
 linkExpirationFormatter.timePeriodUnit.seconds=ثانیه
```

Comment:
Should we add a test to make sure this doesn't break when it's not using a ChoiceFormat?

Reply:
Added another test `format_simple_no_choice` which passes green.